examples/travis-ci-test-and-deploy.yml

```diff
@@ -31,7 +31,11 @@ stages:
   # Only execute deployment stage on master branch, tagged commits,
   # and from your repository (e.g. not PRs). Replace with your repo name.
   - name: deploy
-    if: branch = master AND tag IS PRESENT AND repo = joerick/cibuildwheel
+    if: tag IS PRESENT AND repo = joerick/cibuildwheel
+    # To only build tags on the master branch:
+    #   if: branch = master AND tag IS PRESENT AND repo = joerick/cibuildwheel
```

Comment:
Line 36 wont work through. Why suggest it?

Reply:
Sorry; trying to be too fast and I misread your build log. I thought the build you linked to in #246 was working since it was green, but obviously it isn't!